app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -1,26 +1,172 @@
-<%= presenter.attribute_to_html(:description, html_dl: true, label: t("simple_form.labels.defaults.description")) %>
-<% (Generic.generic_properties.map(&:to_sym)).reject{ |attr| [:description, :language, :license, :oembed_url, :resource_type, :rights_statement].include? attr }.each do |attr| %>
-  <%= presenter.attribute_to_html(attr, html_dl: true, label: t("simple_form.labels.defaults.#{attr}")) %>
-<% end %>
-<%= presenter.attribute_to_html(:language_label, html_dl: true, label: t("simple_form.labels.defaults.language")) %>
-<%= presenter.attribute_to_html(:type_label, html_dl: true, label: t("simple_form.labels.defaults.resource_type")) %>
-<%= presenter.attribute_to_html(:rights_statement_label, html_dl: true, label: t("simple_form.labels.defaults.rights_statement")) %>
-<% Video.video_properties.map(&:to_sym).each do |attr| %>
-  <%= presenter.attribute_to_html(attr, html_dl: true, label: t("simple_form.labels.defaults.#{attr}")) %>
-<% end %>
-<% Document.document_properties.map(&:to_sym).each do |attr| %>
-  <%= presenter.attribute_to_html(attr, html_dl: true, label: t("simple_form.labels.defaults.#{attr}")) %>
-<% end %>
-<% Image.image_properties.map(&:to_sym).each do |attr| %>
-  <%= presenter.attribute_to_html(attr, html_dl: true, label: t("simple_form.labels.defaults.#{attr}")) %>
-<% end %>
+<% @zipped = {} %> 
 <% Generic.controlled_property_labels.map(&:to_sym).each do |attr| %>
   <% labels=Array(presenter.send(attr)) %>
   <% links=Array(presenter.send(attr.to_s.gsub('_label', ''))) %>
-  <% zipped=labels.zip(links) %>
-  <% zipped=Hash[zipped] %>
-  <%= presenter.attribute_to_html(attr, render_as: :search_and_external_link, links: zipped, html_dl: true, label: t("simple_form.labels.defaults.#{attr}")) %>
+  <% @zipped=labels.zip(links) %>
+  <% @zipped=Hash[@zipped] %>
 <% end %>
+
+<%= presenter.attribute_to_html(:trible_title, html_dl: true, label: t("simple_form.labels.defaults.trible_title")) %>
```

Comment:
I wonder if we defined something like `Generic.ordered_properties`, could we repurpose the old loops by just throwing in a check if the property is in the `controlled_property` or `controlled_property_labels`, whichever makes sense.
I don't have a problem with this method, but I think it might be easier to solve the links issue.